Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -126,6 +156,25 @@ final class PledgePaymentPlanOptionView: UIView {
 
         self.delegate?.pledgePaymentPlanOptionView(self, didSelectPlanType: paymentPlan)
       }
+
+    self.viewModel.outputs.notifyDelegateTermsOfUseTapped
+      .observeForUI()
+      .observeValues { [weak self] helpType in
+        guard let self = self else { return }
+
+        self.delegate?.pledgePaymentPlansViewController(self, didTapTermsOfUseWith: helpType)
+      }
+
+    self.termsOfUseButton.rac.hidden = self.viewModel.outputs.termsOfUseButtonHidden
```

Comment:
Yes, we need it because the `termsOfUseButton` is part of the same stack view (`optionDescriptorStackView`) as the `titleLabel`, `subtitleLabel`, `paymentIncrementsStackView`, etc. Any recommendations or thoughts?

<img width="295" alt="image" src="https://github.com/user-attachments/assets/530e9f63-03ce-458f-b003-b66705a9dded" />

Reply:
Ah I thought that there was a stackview just for the items that need to be hidden/shown at the correct time.